padguide2/padguide2.py

```diff
@@ -1132,8 +1132,8 @@ def color_txt_to_list(txt):
             txt = txt.replace('colors', ' ')
             txt = txt.replace('orbs', ' ')
             txt = txt.replace('orb', ' ')
-            txt = txt.replace('critical poison', 'mortal')
-            txt = txt.replace('mortal poison', 'mortal')
+            txt = txt.replace('critical poison', 'mortalpoison')

```

Comment:
Probably don't need both of these cleanups now that we're controlling the text right? I'm sure it's uniformly just one of them but i'm lazy to check which one.